### PR TITLE
Update specs file to change lvs_noheadings

### PR DESCRIPTION
**Pull request by shzhou12**

### CODE REVIEW

File: insights/config/specs.py [hunk 1]
```diff
@@ -199,7 +199,7 @@
     "lvdisplay"                 : CommandSpec("/sbin/lvdisplay"),
     "lvm.conf"                  : SimpleFileSpec("etc/lvm/lvm.conf"),
     "lvs"                       : NoneGroup([CommandSpec('/sbin/lvs -a -o +lv_tags,devices --config="global{locking_type=0}"')]),
-    "lvs_noheadings"            : CommandSpec("/sbin/lvs --nameprefixes --noheadings --separator='|' -a -o lv_name,vg_name,lv_size,region_size,mirror_log,lv_attr,devices,region_size --config=\"global{locking_type=0}\""),
+    "lvs_noheadings"            : CommandSpec("/sbin/lvs --nameprefixes --noheadings --separator='|' -a -o lv_all --config=\"global{locking_type=0}\""),
```
Comment (bfahr) [inline insights/config/specs.py]: @shzhou12 does `-o lv_all` work with older versions of RHEL 6?

Comment (shzhou12) [inline insights/config/specs.py]: @bfahr I tested on rhel6.1.  `-o lv_all` works with it.

**Comment by xiangce** [comment]
@shzhou12 - please re-submit it to the master branch as well

**Comment by xiangce** [comment]
@shzhou12 - It would be better that you add more test cases to the combiner "lvm.py" correspondingly.

File: insights/config/specs.py (hunk 1 above)
Comment (bfahr) [inline insights/config/specs.py]: Thanks @shzhou12 for confirming that.

**Pull request merged**